x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_diff_tab.tsx

```diff
@@ -16,22 +16,77 @@ import {
   EuiTitle,
   EuiIconTip,
 } from '@elastic/eui';
+import { parseDuration } from '@kbn/alerting-plugin/common';
 import type { RuleResponse } from '../../../../../common/api/detection_engine/model/rule_schema/rule_schemas.gen';
 import { DiffView } from './json_diff/diff_view';
 import * as i18n from './json_diff/translations';
 
+/* Inclding these properties in diff display might be confusing to users. */
+const HIDDEN_PROPERTIES = [
+  /*
+    By default, prebuilt rules don't have any actions or exception lists. So if a user has defined actions or exception lists for a rule, it'll show up as diff. This looks confusing as the user might think that their actions and exceptions lists will get removed after the upgrade, which is not the case - they will be preserved.
+  */
+  'actions',
+  'exceptions_list',
+
+  /*
+    Most prebuilt rules are installed as "disabled". Once user enables a rule, it'll show as diff, which doesn't add value.
+  */
+  'enabled',
+
+  /* Technical property. Hidden because it can be confused with "version" by users. */
+  'revision',
+
+  /*
+    This info is not yet exposed by prebuilt rules.
```

Comment:
I would change this explanation. This `updated_at` field is not related to the field that we are trying to add in that ticket. That information is going to be called `elastic_last_update` or something similar and will live within a new `prebuilt` property.

I would just explain, for this case, that `updated_at` is regenerated every time the `/upgrade/_review` endpoint runs and will therefore always show a diff, but it adds no value or make sense for the user.

Reply:
Thanks! Updated!